src/gui/transferlistwidget.cpp

```diff
@@ -896,12 +807,8 @@ void TransferListWidget::displayListMenu(const QPoint &)
     connect(actionDelete, &QAction::triggered, this, &TransferListWidget::softDeleteSelectedTorrents);
     auto *actionPreviewFile = new QAction(UIThemeManager::instance()->getIcon("view-preview"), tr("Preview file..."), listMenu);
     connect(actionPreviewFile, &QAction::triggered, this, &TransferListWidget::previewSelectedTorrents);
-    auto *actionSetMaxRatio = new QAction(UIThemeManager::instance()->getIcon(QLatin1String("ratio")), tr("Limit share ratio..."), listMenu);
-    connect(actionSetMaxRatio, &QAction::triggered, this, &TransferListWidget::setMaxRatioSelectedTorrents);
-    auto *actionSetUploadLimit = new QAction(UIThemeManager::instance()->getIcon("kt-set-max-upload-speed"), tr("Limit upload rate..."), listMenu);
-    connect(actionSetUploadLimit, &QAction::triggered, this, &TransferListWidget::setUpLimitSelectedTorrents);
-    auto *actionSetDownloadLimit = new QAction(UIThemeManager::instance()->getIcon("kt-set-max-download-speed"), tr("Limit download rate..."), listMenu);
-    connect(actionSetDownloadLimit, &QAction::triggered, this, &TransferListWidget::setDlLimitSelectedTorrents);
+    auto *actionTorrentOptions = new QAction(UIThemeManager::instance()->getIcon("configure"), tr("Torrent options"), listMenu);
```

Comment:
imho, "Torrent Options" doesn't seem right here, the dialog that follows doesn't actually include a majority of torrent options.

Reply:
Any suggestions?

Reply:
Anyway menu item that shows dialog should have name with "..." in the end.

Reply:
> Any suggestions?

"Torrent Speed Management" or "Share options", Idk I'm not good with naming.

Reply:
But there may be different types of options. Maybe something less specific, like "Other options... " or "More options..."?

Reply:
I think we should keep the current name.

If you really want to change the name, what about "Torrent **Health** Options"??

Reply:
>what about "Torrent Health Options"??

Because of the same reason  as in https://github.com/qbittorrent/qBittorrent/pull/13885#discussion_r539373673. There may be options of different kind.

Reply:
> 
> 
> Maybe something less specific, like "Other options... " or "More options..."?

To me those titles are for minor/miscellaneous options and I personally don't consider current options (speed,share & DHT/PeX/LSD) as minor for a torrent.

Maybe we should keep the current name??